gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -54,9 +86,26 @@ int main(int argc, char* argv[]) {
     return 1;
   }
 
+  const std::string json_output_dir = absl::GetFlag(FLAGS_json_output_dir);
   const std::string output_dir = absl::GetFlag(FLAGS_output_dir);
-  if (output_dir.empty()) {
-    std::cerr << "Error: --output_dir is required\n";
+  const std::vector<std::string> output_types =
+      absl::GetFlag(FLAGS_output_types);
+  bool json_output_enabled = false;
+  bool asm_output_enabled = false;
+  for (const std::string& output_type : output_types) {
```

Comment:
why do we need to support multiple simultaneous outputs?

Reply:
Mostly a convenience thing. It's nice to be able to reduce test cases by hand using the `llvm-exegesis` binary and it would make it easy to compare against a Gematria benchmarking script. This is especially useful given how slow the Exegesis annotator is. We still want JSON output for the benchmarking script and to limit the number of files on disk for large datasets.